### ISSUE TITLE
Showing new listings for Wednesday, 22 January 2025

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#73]
## Apparent violations of the second law in the quantum-classical dynamics of interacting levitated nanoparticles
- **Keywords:** ['hybrid quantum', 'hybrid quantum']
 - **Authors:** Pedro V. Paraguassú, Thiago Guerreiro
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Statistical Mechanics (cond-mat.stat-mech)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2501.12305
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2501.12305
 - **Abstract**
 Random exchanges of energy arise naturally in stochastic systems. As a consequence, apparent violations of the second law of thermodynamics can occur, as it holds true on average. Here we investigate the occurrence of these apparent violations -- termed free lunches -- in a quantum-classical system comprised of levitated nanoparticles exchanging energy via the Coulomb interaction. We consider different initial states for the quantum system, and how these exert work and fluctuations upon the classical particle affecting the probability of free lunches. With that, we initiate the study of hybrid quantum-classical systems through the lens of stochastic thermodynamics.
## Neural network based deep learning analysis of semiconductor quantum dot qubits for automated control
- **Keywords:** ['semiconductor quantum dot', 'semiconductor quantum dot']
 - **Authors:** Jacob R. Taylor, Sankar Das Sarma
 - **Subjects:** Subjects:
Disordered Systems and Neural Networks (cond-mat.dis-nn); Mesoscale and Nanoscale Physics (cond-mat.mes-hall); Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2405.04524
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2405.04524
 - **Abstract**
 Machine learning offers a largely unexplored avenue for improving noisy disordered devices in physics using automated algorithms. Through simulations that include disorder in physical devices, particularly quantum devices, there is potential to learn about disordered landscapes and subsequently tune devices based on those insights. In this work, we introduce a novel methodology that employs machine learning, specifically convolutional neural networks (CNNs), to discern the disorder landscape in the parameters of the disordered extended Hubbard model underlying the semiconductor quantum dot spin qubit architectures. This technique takes advantage of experimentally obtainable charge stability diagrams from neighboring quantum dot pairs, enabling the CNN to accurately identify disorder in each parameter of the extended Hubbard model. Remarkably, our CNN can process site-specific disorder in Hubbard parameters, including variations in hopping constants, on-site potentials (gate voltages), and both intra-site and inter-site Coulomb terms. This advancement facilitates the prediction of spatially dependent disorder across all parameters simultaneously with high accuracy ($R^2>0.994$) and fewer parameter constraints, marking a significant improvement over previous methods that were focused only on analyzing on-site potentials at low coupling. Furthermore, our approach allows for the tuning of five or more quantum dots at a time, effectively addressing the often-overlooked issue of crosstalk. Not only does our method streamline the tuning process, potentially enabling fully automated adjustments, but it also introduces a "no trust" verification method to rigorously validate the neural network's predictions. Ultimately, this work aims to lay the groundwork for generalizing our method to tackle a broad spectrum of physical problems.
## Entangling power, gate typicality, and measurement-induced phase transitions
- **Keywords:** ['hybrid quantum', 'hybrid quantum']
 - **Authors:** Sourav Manna, Vaibhav Madhok, Arul Lakshminarayan
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Statistical Mechanics (cond-mat.stat-mech); Strongly Correlated Electrons (cond-mat.str-el)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2407.17776
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2407.17776
 - **Abstract**
 When subject to a non-local unitary evolution, qubits in a quantum circuit become increasingly entangled. Conversely, measurements applied to individual qubits lead to their disentanglement from the collective system. The extent of entanglement reduction depends on the frequency of local projective measurements. A delicate balance emerges between unitary evolution, which enhances entanglement, and measurements which diminish it. In the thermodynamic limit, there is a phase transition from volume law entanglement to area law entanglement at a critical value of measurement frequency. This phenomenon, occurring in hybrid quantum circuits with both unitary gates and measurements, is termed as measurement-induced phase transition (MIPT). We study the behavior of MIPT in circuits comprising of two qubit unitary gates parameterized by Cartan decomposition. We show that the entangling power and gate typicality of the two-qubit local unitaries employed in the circuit can be used to explain the behavior of global bipartite entanglement the circuit can sustain. When the two qubit gate throughout the circuit is the identity and measurements are the sole driver of the entanglement behavior, we obtain analytical estimate for the entanglement entropy that shows remarkable agreement with numerical simulations. We also find that the entangling power and gate typicality enable the classification of the two-qubit unitaries by different universality classes of phase transitions that can occur in the hybrid circuit. For all unitaries in a particular universality class, the transition from volume to area law of entanglement occurs with same exponent that characterizes the phase transition.
## The Role of Superlattice Phonons in Charge Localization Across Quantum Dot Arrays
- **Keywords:** ['semiconductor quantum dot']
 - **Authors:** Bokang Hou, Matthew Coley-O'Rourke, Uri Banin, Michael Thoss, Eran Rabani
 - **Subjects:** Subjects:
Chemical Physics (physics.chem-ph); Mesoscale and Nanoscale Physics (cond-mat.mes-hall); Materials Science (cond-mat.mtrl-sci)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2408.02188
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2408.02188
 - **Abstract**
 Understanding charge transport in semiconductor quantum dot (QD) assemblies is important for developing the next generation of solar cells and light-harvesting devices based on QD technology. One of the key factors that governs the transport in such systems is related to the hybridization between the QDs. Recent experiments have successfully synthesized QD molecules, arrays, and assemblies by directly fusing the QDs, with enhanced hybridization leading to high carrier mobilities and coherent band-like electronic transport. In this work, we theoretically investigate the electron transfer dynamics across a finite CdSe-CdS core-shell QD array, considering up to seven interconnected QDs in one dimension. We find that, even in the absence of structural and size disorder, electron transfer can become localized by the emergent low-frequency superlattice vibrational modes when the connecting neck between QDs is narrow. On the other hand, we also identify a regime where the same vibrational modes facilitate coherent electron transport when the connecting necks are wide. Overall, we elucidate the crucial effects of electronic and superlattice symmetries and their couplings when designing high-mobility devices based on QD superlattices.
## Magnon Blockade with Skyrmion Qubit-Magnon Coupling in a Hybrid Quantum System
- **Keywords:** ['hybrid quantum', 'hybrid quantum']
 - **Authors:** Si-Tong Jin (1), Shi-Wen He (1), Zi-Long Yang (1), Xuanxuan Xin (2), Chong Li (1)
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Mesoscale and Nanoscale Physics (cond-mat.mes-hall)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2412.11359
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2412.11359
 - **Abstract**
 Magnon blockade is a fundamental quantum phenomenon for generating single-magnon state, which gradually becomes one of the candidates for quantum information processing. In this paper, we propose a theoretical scheme to generate the magnon blockade in a hybrid system consisting of a YIG micromagnet and a skyrmion. Considering weak probing of the magnon and driving of the skyrmion qubit, the second-order correlation function is analytically derived, and the optimal condition for realizing the magnon blockade is identified. Under the optimal condition, we systematically analyze the behavior of the second-order correlation function $g^{(2)}(0)$ under different parameter regimes. Our analysis shows that with appropriate driving and probing field intensities, the magnon blockade effect can be significantly enhanced, effectively suppressing multi-magnon states and facilitating the generation of high-purity single-magnon states exhibiting pronounced antibunching. Furthermore, we explore the physical mechanisms underlying the magnon blockade, revealing the coexistence and interplay of conventional and unconventional magnon blockade. This scheme provides a versatile all-magnetic platform for generating high purity single-magnon sources.
## Generation of optomicrowave and optomagnonic entanglements in cascaded optomagnomechanical systems
- **Keywords:** ['hybrid quantum']
 - **Authors:** Chong Zhang, Xiaomin Liu, Liwen Gao, Rongguo Yang, Jing Zhang, Tiancai Zhang
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2501.11329
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2501.11329
 - **Abstract**
 The optomagnomechanical system, which involves flexible nonlinearities, is one of the promising physical platforms for studying the preparation and manipulation of quantum entanglements, as well as the construction of hybrid quantum networks. A scheme for entanglement enhancement and quadripartite entanglement generation is proposed, based on a cascaded optomagnomechanical system. On the one hand, optomicrowave and optomagnonic entanglements within the two subsystems are investigated, and their parameter dependence, such as detuning, decay, coupling strength, and transmission efficiency, is discussed. On the other hand, the parameter conditions for achieving optimal optomicrowave and optomagnonic quadripartite entanglements are also obtained. The results show that significant enhancement of optomicrowave and optomagnonic entanglements in the second cavity can be obtained in a certain range of parameters. Under optimized parameter conditions, optomicrowave and optomagnonic quadripartite entanglements can be generated throughout the entire cascaded system. This research provides a theoretical basis for the manipulation of quantum entanglement, the transmission of the magnon's state, and the construction of hybrid quantum networks involving different physical systems.
## Solving Constrained Optimization Problems Using Hybrid Qubit-Qumode Quantum Devices
- **Keywords:** ['hybrid quantum', 'hybrid qubit']
 - **Authors:** Rishab Dutta, Brandon Allen, Nam P. Vu, Chuzhi Xu, Kun Liu, Fei Miao, Bing Wang, Amit Surana, Chen Wang, Yongshan Ding, Victor S. Batista
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2501.11735
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2501.11735
 - **Abstract**
 Optimization challenges span a wide array of fields, from logistics and scheduling to finance, materials science, and drug discovery. Among these, Quadratic Unconstrained Binary Optimization (QUBO) problems are especially significant due to their computational complexity and their potential as a key application for quantum computing. In this work, we introduce an approach for solving QUBO problems using hybrid qubit-qumode bosonic quantum computers$\unicode{x2014}$devices that manipulate and measure the quantum states of light within microwave cavity resonators. We map problems with soft and hard constraints onto the Hamiltonian of a hybrid quantum system, consisting of a single qubit coupled to multiple qumodes. The optimal solution is encoded in the ground state of the system, which is revealed by photon-number measurements. Trial states are prepared through universal qubit-qumode circuits, employing echoed conditional displacement (ECD) gates in combination with qubit rotations. Our approach demonstrates the immense potential of hybrid quantum systems, showcasing their ability to efficiently tackle complex optimization problems in both academia and industry.
## Performance of Superconducting Resonators Suspended on SiN Membranes
- **Keywords:** ['superconducting resonators']
 - **Authors:** Trevor Chistolini, Kyunghoon Lee, Archan Banerjee, Mohammed Alghadeer, Christian Jünger, M. Virginia P. Altoé, Chengyu Song, Sudi Chen, Feng Wang, David I. Santiago, Irfan Siddiqi
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Applied Physics (physics.app-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2405.01784
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2405.01784
 - **Abstract**
 Suspending devices on thin SiN membranes can limit their interaction with the bulk substrate and reduce parasitic capacitance to ground. While suspending devices on membranes is used in many fields including radiation detection using superconducting circuits, there has been less investigation into maximum membrane aspect ratios and achievable suspended device quality, metrics important to establish the applicable scope of the technique. Here, we investigate these metrics by fabricating superconducting coplanar waveguide resonators entirely atop thin ($\sim$110 nm) SiN membranes, where the membrane's shortest length to thickness yields an aspect ratio of approximately $7.4 \times 10^3$. We compare these membrane resonators to on-substrate resonators on the same chip, finding similar internal quality factors $\sim$$10^5$ at single photon levels. Furthermore, we confirm that these membranes do not adversely affect resonator thermalization and conduct further materials characterization. By achieving high quality superconducting circuit devices fully suspended on thin SiN membranes, our results help expand the technique's scope to potential uses including incorporating higher aspect ratio membranes for device suspension and creating larger footprint, high impedance, and high quality devices.
## Conditional Diffusion-based Parameter Generation for Quantum Approximate Optimization Algorithm
- **Keywords:** ['hybrid quantum']
 - **Authors:** Fanxu Meng, Xiangzhen Zhou, Pengcheng Zhu, Yu Luo
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2407.12242
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2407.12242
 - **Abstract**
 The Quantum Approximate Optimization Algorithm (QAOA) is a hybrid quantum-classical algorithm that shows promise in efficiently solving the MaxCut problem, a representative example of combinatorial optimization. However, its effectiveness heavily depends on the parameter optimization pipeline, where the parameter initialization strategy is nontrivial due to the non-convex and complex optimization landscapes characterized by issues with low-quality local minima. Recent inspiration comes from the diffusion of classical neural network parameters, which has demonstrated that neural network training can benefit from generating good initial parameters through diffusion models. Therefore, in this work, we formulate the problem of finding good initial parameters as a generative task and propose the initial parameter generation scheme through dataset-conditioned pre-trained parameter sampling. Concretely, the generative machine learning model, specifically the denoising diffusion probabilistic model (DDPM), is trained to learn the distribution of pretrained parameters conditioned on the graph dataset. Intuitively, our proposed framework aims at effectively distilling knowledge from pre-trained parameters to generate well-performing initial parameters for QAOA. Compared to random parameter initialization, experiments on various-sized Max-Cut problem instances consistently show that our conditional DDPM is capable of improving the approximation ratio by as much as 14.4%, 11.0%, 11.4% and 7.49%, 8.31%, 6.08% on average for random, regular, and Watts-Strogatz graphs, respectively. Additionally, the experimental results also indicate that the conditional DDPM trained on small problem instances can be extrapolated to larger ones, improving the approximation ratio by up to 28.4% and 12.1% on average.
